testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/realm/LoginSettingsTest.java

```diff
@@ -232,6 +232,44 @@ public void rememberMe() {
         
     }
     
+    @Test
+    public void rememberMeWithUtf8Username() {
+        log.info("creating username with non-ASCII username");
+        UserRepresentation utf8TestUser = createUserRepresentation("täst", "test-utf8@email.test", "utf8", "test", true);
+        setPasswordFor(utf8TestUser, PASSWORD);
+
```

Comment:
_UserRepresentation_ is basically DTO, but the user is not created on the server.
Please import this:
```
import static org.junit.Assert.assertNotNull;
import static org.keycloak.testsuite.admin.ApiUtil.createUserWithAdminClient;
```
Create user on the server.
```suggestion
String id = createUserWithAdminClient(testRealmResource(), utf8TestUser);
assertNotNull(id);
```